src/pages/docker/configure/manage-database.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Manage the database
+description: Learn how to access and manage the database in the Cloud Docker for Commerce environment.
+---
+
+# Manage the database
+
+The Cloud Docker development environment provides MySQL services through a MariaDB (default) or MySQL database deployed to the [Docker database container](../containers/service.md#database-container). You connect to the database using `docker-compose` commands. You can also import data from an existing Adobe Commerce on cloud infrastructure project into the database container using the `magento-cloud db:dump` command.
+
+## Connect to the database
+
+You can connect to the database through the Docker container or through the database port. Before you begin, locate the database credentials in the `database` section of the `.docker/config.php` file. The examples use the following default credentials:
+
+> Filename: `.docker/config.php`
```

Comment:
```suggestion
Filename: `.docker/config.php`
```

Reply:
That forces an indent and looks weird. It doesn't format the text.

Reply:
It does. I corrected a few, but clearly there is more out there. I think this was an issue with ExL, somewhat. It was initially used to display a filename before a code block, but here it is clearly awkward. After the merge, I am sure I will dig into Devsite capabilities to present a better experience.